html/demos/streaming.js

```diff
@@ -346,13 +355,9 @@ function updateStreamsList() {
 				// Keep track of all the available streams
 				streamsList[list[mp]["id"]] = list[mp];
 			}
-			$('#streamslist a').unbind('click').click(function() {
-				$('.dropdown-toggle').dropdown('hide');
+			$('#streamslist a').off('click').on('click', function() {
 				selectedStream = $(this).attr("id");
-				$('#streamset').html($(this).html()).parent().removeClass('open');
-				$('#list .dropdown-backdrop').remove();
-				return false;
-
+				$('#streamset').html($(this).html());
```

Comment:
Not sure why you've changed this method. The dropdown stuff was definitely needed, in our previous method, because of UI issues caused by manually messing with the dropdown menu.

Reply:
Nevermind, I checked and it seems ok.

Reply:
PS: I only now noticed you explained why in the description, apologies!